src/zoid/card-fields/component.jsx

```diff
@@ -69,7 +69,8 @@ type CardFieldsExports = {|
     setAttribute : () => void,
     removeAttribute : () => void,
     addClass : () => void,
-    removeClass : () => void
+    removeClass : () => void,
+    getState : () => Object
```

Comment:
```suggestion
    getState : () => ZalgoPromise<Object>
```

Would all of these functions that are exported technically return promises? I'm thinking that every exported function would return a promise since the results won't be immediately available. I may be wrong but that's my understanding based on these unit tests for export functionality in zoid: https://github.com/krakenjs/zoid/blob/3974c52a880e8b7a72201c9ad205b576611e7c65/test/tests/exports.js

Reply:
That makes sense, Greg. Will update this piece!! Thanks for the review